app/views/pieces/show.html.erb

```diff
@@ -1,32 +1 @@
-<h1> WELCOME TO YOUR GAME MWAHAHAHAH </h1>

```

Comment:
@JefeNelson if you delete the show action from the routes and controller then you should go ahead and delete the view page too since no one will be able to view